src/main/java/seedu/duke/Ui.java

```diff
@@ -300,27 +309,27 @@ public void displayOnePropertyShort(Property property, int i) {
         System.out.println(LINE_BREAK);
     }
 
-    public void displayNoOfProperties(int i) {
-        if (i == 1) {
-            System.out.println("There is " + i + " property in the list");
+    public void displayNoOfProperties(int i, boolean isOneProperty) {
+        if (isOneProperty) {
+            System.out.println(THERE_IS + i + PROPERTY_SINGULAR + IN_THIS_LIST);
         } else {
-            System.out.println("There are " + i + " properties in this list");
+            System.out.println(THERE_ARE + i + PROPERTIES_PLURAL + IN_THIS_LIST);
```

Comment:
You could try changing the message to have an optional 's', e.g. client(s) ,so that you don't need extra code to handle plural and singular list results